src/application/WorkoutSelectionViewController.java

```diff
@@ -199,11 +199,103 @@ void storeSets(Scene exerciseSelectionScene, ExerciseSets exercise, ArrayList<Te
     	applicationStage.setScene(exerciseSelectionScene);
     }
     
+    double calculatedBMI = 0;
+    boolean isMetric = true;
+    Label calculatedBMILabel = new Label();

```

Comment:
We shouldn't have non private/encapsulated instance variables in the controller class (or any class), and the calculate BMI method should really be separated into it's own class